package.json

```diff
@@ -13,14 +13,14 @@
   "dependencies": {
     "aws-sdk": "^2.720.0",
     "cfn-stack-event-stream": "^1.0.1",
-    "colors": "^1.3.1",
+    "colors": "1.4.0",
     "cuid": "^2.1.1",
     "d3-queue": "^3.0.2",
     "diff": "^3.5.0",
     "easy-table": "^1.0.0",
     "fasterror": "^1.0.0",
-    "inquirer": "^6.1.0",
-    "json-diff": "^0.5.2",
+    "inquirer": "^8.2.0",
+    "json-diff": "^0.7.1",
```

Comment:
Can we keep this PR exclusive to colors and have separate ones to update other packages. Or did you do this for another reason?

Reply:
It's been historically difficult to get things merged into this repo & npm audit was throwing a warning for both of these. An upstream client of DevelopmentSeed's had problems with their security compliance department due to the npm audit flag and as such I had bundled them together.

Reply:
ok, for now I think it is fine, if we have any issues we might roll back.